docs/glossary.md

```diff
@@ -0,0 +1,344 @@
+## A
```

Comment:
Could you add a title to the page please, similar to line 1 in the other .md files in the repo.

I know mkdocs adds a top level header automatically, but specifying our own maintains consistency with all the other pages, and also allows us to add an icon to the header (usually supplied by @mariapleshkova or @linnutee)

Reply:
Added title. Just going with no icon for now, similar to the Home page. But would be cool to have an icon.

Also, put a line below the title, as also appears standard on the other pages.